docs/source/index.rst

```diff
@@ -94,7 +94,7 @@ Workflows can be customised using some convenient setter functions. First, we ne
 
 .. code-block:: python
 
-   from openqaoa.devices import create_device
+   from openqaoa.backends import create_device
```

Comment:
The new import is `   from openqaoa import create_device`. But I will fix it directly